internal/services/contracts_events_consumer.go

```diff
@@ -145,3 +153,51 @@ func (c *ContractsEventsConsumer) setPrivilegeHandler(e *ContractEventData) erro
 
 	return nil
 }
+
+type MintedAftermarketDeviceArgs struct {
+	TokenID       string `mapstructure:"newTokenId"`
+	DeviceAddress string
+	Owner         string // TODO - confirm who the minter is
```

Comment:
I'm a little more worried about checking that the contract is what we think it should be. The owner is a bit of a weird thing here.